plugins/georeplication/gsyncd.go

```diff
@@ -37,7 +37,7 @@ func (g *Gsyncd) Path() string {
 
 // Args returns arguments to be passed to gsyncd process during spawn.
 func (g *Gsyncd) Args() []string {
-	g.args = []string{}
+	g.args = make([]string, 0, 6)
```

Comment:
Use a specific size only if it won't change in future or if length is computed using `len()`. The args here may change in future, might become 5 or 7 which requires this to be changed. So, please don't do this

`premature optimization is the root of all evil`

Reply:
done